app/models/account_request.rb

```diff
@@ -48,6 +48,22 @@ def processed?
     organization.present?
   end
 
+  def approved?
+    "Approved" if confirmed? && AccountRequest.includes([:organization])
```

Comment:
I chose to add `AccountRequest.includes([:organization])` here and on line `56` to appease a `bullet` warning:
<img width="761" alt="Screen Shot 2021-09-02 at 11 06 59 AM" src="https://user-images.githubusercontent.com/32834804/131928975-04ebb98c-fd59-4fc2-8c77-3a8c7b4f9903.png">

Reply:
@juliannatetreault does it make more sense to put the `includes` in the controller when accessing all the `AccountRequest` records? I haven't seen this approach done before.

Reply:
I'm actually unsure about that, @edwinthinks -- I took this approach due to the `bullet` warning. 🤔

Reply:
@juliannatetreault sorry for the late reply. You can actually setup the eager loading in the controller. So I would add the `includes` statement in app/controllers/admin/account_requests_controller.rb in the `index` method. So something like:
```
  def index
    @open_account_requests = AccountRequest.includes(:organization).where(confirmed_at: nil).order('created_at DESC')
    @closed_account_requests =  AccountRequest.includes(:organization).where.not(confirmed_at: nil).order('confirmed_at DESC')
  end
```

Reply:
No worries and likewise, @edwinthinks! I will make this change this week. Thanks for the help. 😊

Reply:
Hey there @juliannatetreault , I noticed that the changes we discuss aren't in the PR. Was wondering if you didn't push up the commit with the changes?

Reply:
I just pushed up the most recent, working changes that I made, @edwinthinks -- sorry for the delay! 

This being said though, I was unable to set the eager loading up in the `AccountRequestsController` as discussed, and instead took a different approach in the `AccountRequest` model itself that appeased the Bullet warnings--let me know what you think! I'm happy to make any further adjustments if necessary.